pkg/kubectl/cmd/drain.go

```diff
@@ -51,7 +50,7 @@ import (
 )
 
 type DrainOptions struct {
-	client             internalclientset.Interface
+	clientSet          *kubernetes.Clientset
```

Comment:
nit: we often call it client although it's a clientset. clientSet in fact looks a bit unusual.

Reply:
sure, will fix it